src/controllers/AdminController.php

```diff
@@ -1,16 +1,16 @@
 <?php namespace Frozennode\Administrator;
 
-use Illuminate\Support\Facades\App;
-use Illuminate\Support\Facades\Config;
+use App;
+use Config;
 use AdministratorBaseController as Controller;
-use Illuminate\Support\Facades\Redirect;
-use Illuminate\Support\Facades\Response;
-use Illuminate\Support\Facades\Request;
-use Illuminate\Support\Facades\View;
-use Illuminate\Support\Facades\Input;
-use Illuminate\Support\Facades\File;
-use Illuminate\Support\Facades\Session;
-use Illuminate\Support\Facades\URL;
+use Redirect;
+use Response;
+use Request;
+use View;
+use Input;
+use File;
+use Session;
+use URL;
```

Comment:
Let's make sure we use the full facade classes here. The reason why I enumerated them was because users sometimes change their app config aliases to something else.